tests/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+sudo apt-get update -y
+sudo apt-get install -y bats
+
+# kubectl
+sudo apt-get install -y apt-transport-https ca-certificates curl
+sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+sudo apt-get update -y
+sudo apt-get install -y kubectl
+
+# install devspace
+curl -s -L "https://github.com/loft-sh/devspace/releases/latest" | sed -nE 's!.*"([^"]*devspace-linux-amd64)".*!https://github.com\1!p' | xargs -n 1 curl -L -o devspace && chmod +x devspace;
+
+# configure locales
+sudo apt install -y locales
+echo "LC_ALL='en_US.UTF-8'" | sudo tee /etc/environment
+sudo localedef -i en_US -f UTF-8 en_US.UTF-8
```

Comment:
⛏️ If we already setup the testing environment, it might be better to get the `detik` bash files at this moment too rather than "vendoring" them in the repo like this (it will avoid a potential mess as thinking we "manage" these files).

https://github.com/bats-core/bats-detik#manual-setup